src/main/java/org/minions/devfund/carlosorellana/WhackAMole.java

```diff
@@ -0,0 +1,168 @@
+package org.minions.devfund.carlosorellana;
+
+import java.util.Arrays;
+
+/**
+ * The goal is create backend the WhackAMole.
+ *
+ * @author Carlos Orellana.
+ * @since 07/10/2018.
+ */
+public class WhackAMole {
+
+    private int score;
+    private int molesLeft;
+    private int attemptsLeft;
+    private char[][] moleGrid;
+
+    private static final String SPACE = " ";
+    private static final String NEW_LINE = "\n";
+    private static final char MOLE_CHAR = 'M';
+    private static final char NO_MOLE_CHAR = '*';
+    private static final char WHACKED_CHAR = 'W';
+
+    private String messageAttempts;
+
+    /**
+     * Constructor WhackAMole class.
+     *
+     * @param numAttempts - Number Attempts.
+     * @param gridDimension - Grid dimensions.
+     */
+    public WhackAMole(final int numAttempts, final int gridDimension) {
+        this.attemptsLeft = numAttempts;
+        this.moleGrid = new char[gridDimension][gridDimension];
+
+        for (char[] mole : this.moleGrid) {
+            Arrays.fill(mole, NO_MOLE_CHAR);
+        }
+    }
+
+    /**
+     * Sends the Mole positions in the Grid.
+     *
+     * @param x - X positions.
+     * @param y - Y positions.
+     * @return True if the position is correct, otherwise false.
+     */
+    public boolean place(final int x, final int y) {
+        int xPosition = x - 1;
+        int yPosition = y - 1;
+
+        if (isCorrectPosition(xPosition, yPosition)) {
+            if (moleGrid[xPosition][yPosition] != MOLE_CHAR) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)